front/lib/models/assistant/agent.ts

```diff
@@ -161,19 +164,34 @@ AgentConfiguration.init(
       type: DataTypes.TEXT,
       allowNull: false,
     },
+    pictureUrl: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
     instructions: {
       type: DataTypes.TEXT,
       allowNull: true,
     },
+    providerId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      defaultValue: DEFAULT_ASSISTANT_STATE.modelConfiguration.providerId,
```

Comment:
Can we add TODOs to remove this ? IMO this should be temporary while you migrate but should quickly be removed.